vendor/assets/college_green/mixins.scss

```diff
@@ -0,0 +1,156 @@
+@mixin text-shadow($string: 0 1px 3px rgba(0, 0, 0, 0.25)) {
```

Comment:
It would be better to add [autoprefixer-rails](https://github.com/ai/autoprefixer-rails) and drop all these mixins. Doing that also has the advantage your own sass will be properly prefixed!